### PR TITLE
Add website

### PR DESCRIPTION
**Pull request by Noviny**
WIP - add website

Done:
- Can display documentation, examples, guides
- some coloring done
- Lots of pairing back

To-do
- pull examples into the website structure, out of storybook
- Provide wrapper for examples so they do not look awful (header, code view etc)
- break up readme into docs sections
- add about section information
- solve for content both in readme and website without duplicating it
- continue to tear the template apart, pulling out bits we are not using/do not want
- Fix flow and eslint for template code

### CODE REVIEW

**Comment by Noviny** [comment]
@alexreardon Now advanced enough I would be comfortable merging it to get the examples into a single location so we don't get out of sync.

Netlify deploy failed because stories are removed. What do you want to do to resolve this?

**Comment by alexreardon** [comment]
Let's chat in the office when we are both in :)

File: website/.codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+engines:
```
Comment (alexreardon) [inline website/.codeclimate.yml]: do we need this?

File: website/.browserslistrc [hunk 2]
```diff
@@ -0,0 +1 @@
+Last 2 versions
```
Comment (alexreardon) [inline website/.browserslistrc]: can we please use the same browserlist as the root?

File: website/content/README.md [hunk 3]
```diff
@@ -0,0 +1,1471 @@
+[![Build Status](https://travis-ci.org/atlassian/react-beautiful-dnd.svg?branch=master)](https://travis-ci.org/atlassian/react-beautiful-dnd) [![npm](https://img.shields.io/npm/v/react-beautiful-dnd.svg)](https://www.npmjs.com/package/react-beautiful-dnd) [![dependencies](https://david-dm.org/atlassian/react-beautiful-dnd.svg)](https://david-dm.org/atlassian/react-beautiful-dnd) [![Greenkeeper badge](https://badges.greenkeeper.io/atlassian/react-beautiful-dnd.svg)](https://greenkeeper.io/) [![SemVer](https://img.shields.io/badge/SemVer-2.0.0-brightgreen.svg)](http://semver.org/spec/v2.0.0.html)
```
Comment (alexreardon) [inline website/content/README.md]: Can we not commit this and just have a TODO here for now?

File: website/jsconfig.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (alexreardon) [inline website/jsconfig.json]: what is this file for?

File: website/package.json [hunk 5]
```diff
@@ -0,0 +1,67 @@
+{
+  "name": "gatsby-docs-starter",
```
Comment (alexreardon) [inline website/package.json]: can we update the properties in this?

File: website/src/layouts/css/index.css [hunk 6]
```diff
@@ -0,0 +1,63 @@
+/**********
```
Comment (alexreardon) [inline website/src/layouts/css/index.css]: Interesting! I might look at using the atlaskit reset but that can wait

File: website/package.json (hunk 5 above)
Comment (Noviny) [reply website/package.json]: Should we? The website is effectively its own project, and understanding where it came from to make it easier to chase down seems useful.

Comment (alexreardon) [reply website/package.json]: Yep, but it is our project. So we can change the name to `react-beautiful-dnd-docs` 👍

File: website/.babelrc [hunk 7]
```diff
@@ -0,0 +1,3 @@
+{
+    "plugins": ["lodash"]
```
Comment (alexreardon) [inline website/.babelrc]: can we remove this now that we are not using lodash?
also can you remove the plugin (and this whole file?)

File: website/static/gatsby-advanced-starter [hunk 8]
```diff
@@ -0,0 +1 @@
+.
```
Comment (alexreardon) [inline website/static/gatsby-advanced-starter]: can we remove this file?

**Pull request merged**